ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.14.yaml

```diff
@@ -154,6 +154,16 @@ tests:
       COMPUTE_NODE_TYPE: c5n.metal
       CONTROL_PLANE_INSTANCE_TYPE: m4.xlarge
     workflow: hypershift-kubevirt-e2e
+- as: e2e-kubevirt-aws-csi
+  optional: true
+  steps:
+    cluster_profile: aws
+    env:
+      CNV_PRERELEASE_VERSION: "4.13"
+      COMPUTE_NODE_REPLICAS: "1"
+      COMPUTE_NODE_TYPE: c5n.metal
+      CONTROL_PLANE_INSTANCE_TYPE: m4.xlarge
+    workflow: hypershift-kubevirt-csi-e2e
```

Comment:
you don't need to add this to the 4.14 or any other release configs. They are automatically synced i believe with the fast forward rules for the latest dev branch

Reply:
Removed